### PR TITLE
Fulpstation Cat

### PR DESCRIPTION
**Pull request by Fingusfulp**
Adds a unique pastry cat to fulpstation, complete with crafting menu recipe

Why its good for the game: more fulpstation community easter eggs

### CODE REVIEW

File: code/FulpstationCode/fulp_mobs/eclatmob.dm [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/mob/living/simple_animal/pet/cat/eclat //for a good pal, Fingus
```
Comment (ATH1909) [inline code/FulpstationCode/fulp_mobs/eclatmob.dm]: this should be a subtype of a cak

Comment (remove32) [reply code/FulpstationCode/fulp_mobs/eclatmob.dm]: very much this - its almost the same thing as the cak

**Comment by remove32** [comment]
image
when you dont know how to use git 😜

**Comment by Fingusfulp** [comment]
> image
> when you dont know how to use git 😜

its very true

File: code/FulpstationCode/fulp_mobs/eclatmob.dm [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	desc = "It's a cream filled cat"
+	icon = 'icons/Fulpicons/fulp_mob_icons/eclat.dmi'
+	icon_state = "eclat"
+	icon_living = "eclat"
+	icon_dead = "eclat_dead"
+
+	health = 69
+	maxHealth = 69
+	gender = FEMALE
+	harm_intent_damage = 10
```
Comment (ATH1909) [inline code/FulpstationCode/fulp_mobs/eclatmob.dm]:
```suggestion

```
this is already inherited from caks

File: code/FulpstationCode/fulp_mobs/eclatmob.dm [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	butcher_results = list(/obj/item/organ/brain = 1, /obj/item/organ/heart = 1, /obj/item/reagent_containers/food/snacks/donut/choco = 2,  \
+	/obj/item/reagent_containers/food/snacks/meat/slab = 2)
+	response_harm_continuous = "takes a bite out of"
+	response_harm_simple = "take a bite out of"
+	attacked_sound = 'sound/items/eatfood.ogg'
+	deathmessage = "loses its false life and collapses!"
+	deathsound = "bodyfall"
```
Comment (ATH1909) [inline code/FulpstationCode/fulp_mobs/eclatmob.dm]:
```suggestion

```
this is already inherited from caks

File: code/FulpstationCode/fulp_mobs/eclatmob.dm [hunk 4]
```diff
@@ -0,0 +1,20 @@
+/mob/living/simple_animal/pet/cat/cak/eclat //for a good pal, Fingus
+	name = "Eclair"
+	desc = "It's a cream filled cat"
```
Comment (ATH1909) [inline code/FulpstationCode/fulp_mobs/eclatmob.dm]:
```suggestion
	desc = "It's a cream filled cat."

```

**Comment by JohnFulpWillard** [comment]
uh oh its stale again

**Pull request closed without merging**